models/model.js

```diff
@@ -16,9 +21,19 @@ const fs = require('fs/promises');
         
 
     }
+   async function selectArticle(articleId){
+    if(await checkIfIdExists('articles',articleId,'article_id')){
```

Comment:
I'm not totally against the idea of creating a check if exists function.
But please see my other comment about SQL injection.
And do we need one or do we get a response from the database which we can use to tell us if an Article ID exists or not?